.github/workflows/build-branch.yml

```diff
@@ -3,14 +3,13 @@ name: Build Branch SNAPSHOT
 on:
   workflow_dispatch:
   push:
-    branches-ignore:
+    branches:
```

Comment:
Doesn't this file https://github.com/liquibase/liquibase/blob/master/.github/workflows/build-main.yml cover the master build? Doesn't that make this a duplicate of that build?

Reply:
you are right. Run `master` on every push to `build-main` and run `build-branch` on every `pull_request_target` types specified